index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+    
```

Comment:
Please avoid skipping lines when typing code. This is not considered a good Html practice and also not skipping a line when typing your code makes it look smarter and cleaner.